### ISSUE TITLE
Ghost hound color

### ISSUE THREAD

**Issue by ToBeFree** [arianne/stendhal#65]
Hey :) Just a quick thought. The new ghost hound is not red, but it drops carbuncles. Should we change drops or color maybe?

**Comment by AntumDeluge** [comment]
I forgot that they drop carbuncles. I made a red version just in case. We should probably just switch to that one.

**Comment by AntumDeluge** [comment]
Or, we could use the red image to create hell hounds that drop carbuncle. And put them down in hell.

**Comment by ToBeFree** [comment]
I love the idea of a second hound! But please don't restrict it to hell. 😄

**Comment by AntumDeluge** [comment]
Actually, I don't think that's a bad idea. I was already thinking of putting the hell hound in an area that can only be reached if a condition is met. Maybe killing the ghost hound could be that condition.

If it weren't only in hell, what other maps would it be on?

**Comment by ToBeFree** [comment]
It could be hiding in an old mine house that currently has no door, and to get in, you'd need a key the ghost hound drops. This house could then also be a secret meeting point for all those who have already killed a ghost hound. ^^

**Comment by ToBeFree** [comment]
The top of 0_semos_mountain_n2_e has a large completely unused house ruin that could be locked or contain a trapdoor downstairs

**Comment by AntumDeluge** [comment]
That actually might be the house that I am using for a quest that is in the works. If it is, it already has a map for the interior. I'll have to check it out.

**Comment by AntumDeluge** [comment]
Nope, it's not the same house. The one I am thinking of is a little shack in 0_semos_mountain_n2_w.

**Comment by ToBeFree** [comment]
😃 

(The little shack looks awesome for a new interior map!)

**Comment by AntumDeluge** [comment]
I don't think the new *hell hound* creature will make it into the next release. Should I change *ghost hound* back to red for now... until the red *hell hound* is implemented?

**Comment by ToBeFree** [comment]
(late comment: both are fine, I'd say!)
